docs/src/almalinux_instructions.rst

```diff
@@ -0,0 +1,257 @@
+.. _almalinux_instructions:
+
+Step-by-step instructions for Almalinux 8.4
+===========================================
+
+.. note::
+
+    These instructions should also work for RHEL8, CentOS8, and Rocky Linux. Some changes may be necessary for RHEL7/CenOS7.
+
+
+1. Install Node.js
+~~~~~~~~~~~~~~~~~~
+
+carta-controller uses `Node.js <https://nodejs.org/>`_ and supports v12, v14, and v16. Node.js can easily be installed from the AlmaLinux AppStream repository. Here we install v14, as well as the ``npm`` package manager.
+
+.. code-block:: shell
+
+    # Install Node.js v14:
+    sudo dnf module enable nodejs:14
+    sudo dnf install -y nodejs npm
+
+    # Check it is installed and working:
+    node --version
+    npm --version
+
+2. Install MongoDB
+~~~~~~~~~~~~~~~~~~
+
+carta-controller uses `MongoDB <https://www.mongodb.com/>`_ to store user preferences etc. MongoDB is not available through the default AlmaLinux repositories, but we can create to custom repository file in order to easily install it.
```

Comment:
"preferences etc." -> "preferences, etc.."
"we can create to custom repository file in order to easily install it" -> "we can add a custom repository to install it more easily"